### ISSUE TITLE
Combine plugin update pages

### ISSUE THREAD

**Issue by onli** [s9y/Serendipity#383]
Currently we have one page for updating sidebar plugins and another for updating event plugins. I argue that this difference is coming from how the plugin api developed and how the code looks like, but it is not an useful separation for users. We should combine updates on one page.

Discussion: http://board.s9y.org/viewtopic.php?f=11&t=20630

**Comment by onli** [comment]
I'm currently fighting with my system and thus have no proper editor. I would need someone to set PLUGIN_EVENT_SPARTACUS_CHECK to "Update plugins" in **serendipity_event_spartacus/lang_en.inc.php**. I think I'd run into encoding problems if I tried that with my current editor.

**Comment by yellowled** [comment]
@onli Done.

**Comment by onli** [comment]
`serendipity[spartacus_fetch]=` is still a problem. That's in the link triggering the upgrade, and needs to get rewritten to not use the one baseURL set by spartacus (event|sidebar), but one specific to the plugin type. Basically just moving it into `$pluginstack`, but I think I run into caching issues there.

**Comment by onli** [comment]
The link is now fixed. I tested it, this time with a bit more care, and saw both types of plugins really being upgraded.

9d18a74 is not the nicest approach. It would be better to let spartacus set the baseURL. But I really did not find out why the plugin count varies, and if the backend does not go through all plugin classes we can't realiably move spartacus baseURI into the objects in `$pluginstack`. In practice it should not matter anyway, spartacus has no alternative.

Would be happy if someone could test and close this.
